vision/blob/blobfind.py

```diff
@@ -0,0 +1,75 @@
+import cv2
+import numpy as np
+import vision.blob.constants as constants
+from vision.blob.blob import Rectangle
+import os
+
+def configure_params():
+    params = cv2.SimpleBlobDetector_Params()
+
+    params.filterByArea = True
+    params.maxArea = constants.MAX_AREA
+
+    params.minThreshold = constants.MIN_THRESHOLD
+    params.maxThreshold = constants.MAX_THRESHOLD
+
+    return params
```

Comment:
Lets make the config a .json so we dont need a custom parser and itll be more accessible to others trying to use the config or change it. See my comments on the constants.py.